src/PointwiseFunctions/AnalyticData/GrMhd/InitialMagneticFields/Poloidal.cpp

```diff
@@ -77,26 +94,25 @@ Poloidal::variables(const tnsr::I<DataType, 3>& coords,
         pow(pressure_i - cutoff_pressure_,
             static_cast<int>(pressure_exponent_) - 1);
 
-    const auto& dp_dx = get_element(dcoords_pressure.get(0), i);
-    const auto& dp_dy = get_element(dcoords_pressure.get(1), i);
-    const auto& dp_dz = get_element(dcoords_pressure.get(2), i);
+    const auto& dp_dx = get_element(deriv_pressure.get(0), i);
+    const auto& dp_dy = get_element(deriv_pressure.get(1), i);
+    const auto& dp_dz = get_element(deriv_pressure.get(2), i);
 
     // Assign Bx, By, Bz
-    get_element(magnetic_field.get(0), i) =
-        -n_times_pressure_to_n_minus_1 * x * dp_dz;
-    get_element(magnetic_field.get(1), i) =
-        -n_times_pressure_to_n_minus_1 * y * dp_dz;
-    get_element(magnetic_field.get(2), i) =
-        2.0 * pressure_term +
-        n_times_pressure_to_n_minus_1 * (x * dp_dx + y * dp_dy);
+    get_element(magnetic_field->get(0), i) =
```

Comment:
This overwrites instead of adding unlike all the other changes.  I'm guessing you want `+=` to be consistent with your other changes; otherwise I am confused....